app/styles/ui/_cloning-repository-view.scss

```diff
@@ -66,12 +66,13 @@
     overflow: hidden;
```

Comment:
```suggestion
    overflow: 

> `hidden;`

```

Reply:
hello @memodrnn , i couldn't get you. a little explanation would be helpful

Reply:
Don't worry about that comment @maifeeulasad, it was just a spammy message 😓